src/ViewRequest.js

```diff
@@ -291,8 +291,8 @@ class ViewRequest extends React.Component {
         lastMenu={detailMenu}
         actionMenuItems={!isRequestClosed ? [{
           id: 'clickable-edit-request',
-          title: 'Edit Request',
-          label: 'Edit',
+          title: formatMessage({ id: 'ui-requests.actions.editRequest' }),
```

Comment:
@MaksymDryha is there a reason we can't use `<FormattedMessage>` here?

Reply:
The reason I didn't use `<FormattedMessage>` here was that `actionMenuItems` is an array with more than one title string. Wrapping `<Pane>` with `<FormattedMessage>`, would only allow us to translate one string so I decide to use `formatMessage` separetly for each title.
But based on @cherewaty's comment about `title`, this is no longer relevant. I removed those attributes.